src/main/java/frc/robot/subsystems/Indexer.java

```diff
@@ -49,22 +48,28 @@ public Indexer(final IndexMap map) {
 
     }
 
-    public SequentialCommandGroup intakeToPierre() {
-        return new SequentialCommandGroup(pierrePossesion(), runToClearBottomSensor());
+    public CommandBase intakeToPierre() {
+        CommandBase cmd = new SequentialCommandGroup(pierrePossesion(), runToClearBottomSensor());
+        cmd.setName("Intake to Pierre");
+        return cmd;
     }
 
-    public SequentialCommandGroup shootOneBall() {
-        return new SequentialCommandGroup(loadBallToTop(), unLoadBall());
+    public CommandBase shootingBalls() {
```

Comment:
Why is it called `shootingBalls`? why the `ing`? Why plural balls?

Reply:
Don't Know *cough* *cough* @Rohan-Sreenivasan

Reply:
*Fixed